docs/platforms/dotnet/common/crons/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Set Up Crons
+description: "Learn how to enable Cron Monitoring in your app"
+sidebar_order: 5000
+---
+
+Sentry Crons allows you to monitor the uptime and performance of any scheduled, recurring job. Once implemented, it'll allow you to get alerts and metrics to help you solve errors, detect timeouts, and prevent disruptions to your service.
+
+## Requirements
+
+<PlatformContent includePath="crons/requirements" />
+
+<PlatformContent includePath="crons/setup" />
+
+## Alerts
+
+When your recurring job fails to check in (missed), runs beyond its configured maximum runtime (failed), or manually reports a failure, Sentry will create an error event with a tag to your monitor.
+
+To receive alerts about these events:
+
+1. Navigate to **Alerts** in the sidebar.
+2. Create a new alert and select "Issues" under "Errors" as the alert type.
```

Comment:
This seem like product docs as opposed to platform specific.

Was this is copied from another crons docs? I feel if we change this we'll have a lot of outdated docs.
Shoudl these steps be on https://docs.sentry.io/product/alerts/#creating-alerts instead?

Perhaps specifically for Crons and we link from here, and from the Crons product page?

Reply:
Looks like all the platform crons docs have this too. I agree that it's more product specific and should be moved to the alerts page you linked, but we'd also need to update the other platform docs to link there.